metadb/service.go

```diff
@@ -0,0 +1,41 @@
+package metadb
+
+import (
+	"context"
+
+	"github.com/pastelnetwork/gonode/metadb/rqlite/store"
+)
+
+const (
+	logPrefix = "metadb"
+)
+
+// Service represents the rqlite cluster
+type Service interface {
+	// Run starts the rqlite service
+	Run(ctx context.Context) error
+	// Query execute a query, not support multple statements
+	// level can be 'none', 'weak', and 'strong'
+	Query(ctx context.Context, statement string, level string) (*QueryResult, error)
+	// Write execute a statement, not support multple statements
+	Write(ctx context.Context, statement string) (*WriteResult, error)
+}
```

Comment:
Oh, I have should explained it in the previous reviews.

The `metadb` package must provide DAO (data access object) instead of sql query executor. 
Business logic services do not need to know anything about how and where data is stored, and even more so, they do not need to compose sql queries. If tomorrow we change the type of db, we should not touch the code of the business services.

It should look something like this:
```go
StoreArtworksData(ctx context.Context,  data *SomeDataStruct) error
GetArtworksData(ctx context.Context,  txid string) (*SomeDataStruct, error)
```
But it's just example, I don't know exactly what we are going to store, in details. To clarify this, ask Jeff or aik

Reply:
yeah, it should be the next task, i will ask @jeff and @ayk

Reply:
Yes please do it, you can create the task yourself in Freedcamp.

Reply:
@Dicklesworthstone already create a task